ckan/lib/helpers.py

```diff
@@ -2344,6 +2344,35 @@ def mail_to(email_address, name):
     return html
 
 
+@core_helper
+def get_base_public_folder(app_conf):
```

Comment:
ckan/lib/helpers are functions for templates to call. It makes no sense for a template to need a directory, could we move these closer to where they're actually needed?